### DISCUSSION TITLE
,

### DISCUSSION THREAD

**Discussion by Jekyle9444** [microsoft/vscode-discussions#2460] [Extension Show and Tell]
Start

**Comment by alefragnani** [comment]
Hi @Jekyle9444 ,

Could you please elaborate your question?

Also, please take a look at https://github.com/microsoft/vscode-discussions/discussions/5. If your question is not about extension development, but VS Code usage, the right place to post would be the VS Code repo (https://github.com/microsoft/vscode).

Hope this helps
